### PR TITLE
Remove file read and write from node lib

### PR DESCRIPTION
**Pull request by DarylThayil**
Library gets json and outputs json

Commented out sync stuff for out, but this maybe used when beforeCacheAccess and afterCacheAccess are implemented

### CODE REVIEW

File: lib/msal-node/src/cache/CacheContext.ts [hunk 1]
```diff
@@ -26,54 +22,25 @@ export class CacheContext {
...
-    }
-
-    /**
-     * sets the cache path provided by the user
-     * @param path
-     */
-    setCachePath(path: string) {
-        this.cachePath = path;
-    }
+    constructor() {}
```
Comment (sangonzal) [inline lib/msal-node/src/cache/CacheContext.ts:25]: Is this needed?

Comment (sameerag) [reply lib/msal-node/src/cache/CacheContext.ts:25]: if you are removing `cachePath`, we no longer need a constructor IMO.

Comment (jasonnutter) [reply lib/msal-node/src/cache/CacheContext.ts:25]: Empty constructor is not needed.

File: samples/msal-node-auth-code/index.js [hunk 2]
```diff
@@ -51,11 +53,12 @@ app.get('/redirect', (req, res) => {
...
     pca.acquireTokenByCode(tokenRequest).then((response) => {
         console.log("\nResponse: \n:", response);
-        // console.log(pca.getCache());
         res.send(200);
-        pca.writeCacheToDisk("/Users/sameeragajjarapu/Documents/cache.json");
+        // uncomment this to show writing of cache, dont commit real tokens.
+        // fs.writeFileSync("./data/cache.json", JSON.stringify(pca.readCache()), null, 4);
     }).catch((error) => {
-        res.send(500);
+        console.log(error);
```
Comment (jasonnutter) [inline samples/msal-node-auth-code/index.js:60]: Nit: `console.error`

File: samples/msal-node-auth-code/index.js [hunk 3]
```diff
@@ -51,11 +53,12 @@ app.get('/redirect', (req, res) => {
 
     pca.acquireTokenByCode(tokenRequest).then((response) => {
         console.log("\nResponse: \n:", response);
-        // console.log(pca.getCache());
         res.send(200);
-        pca.writeCacheToDisk("/Users/sameeragajjarapu/Documents/cache.json");
+        // uncomment this to show writing of cache, dont commit real tokens.
+        // fs.writeFileSync("./data/cache.json", JSON.stringify(pca.readCache()), null, 4);
```
Comment (jasonnutter) [inline samples/msal-node-auth-code/index.js:58]: Use the `async` version (preferably the promise-based one)

**Pull request merged**